src/deploy/NVA_build/logrotate_noobaa.conf

```diff
@@ -30,8 +30,9 @@
 
 /var/log/bucket_logs.log
 {
-        daily
-        size 100k
+        hourly
```

Comment:
hourly check is needed?

Reply:
@naveenpaul1 yes, Actually these logs should be uploaded to log buckets and which would be consumed by other process. Its not only when some bug happens. These are logs for bucket operations.